application/classes/Controller/Api/CSV/Import.php

```diff
@@ -29,6 +29,12 @@ protected function _resource()
 
 	public function action_post_index_collection()
 	{
+		/**
+		 * Step two of import.
+		 * Support all line endings without manually specifying it (primarily added because of OS9 line endings which do not work by default )
+		 */
+		ini_set('auto_detect_line_endings', 1);
```

Comment:
Is there a downside to setting this option in the global ini? We can do this via ansible config. Or is there a reason we want this setting only during the execution cycle of the import?

Reply:
Yeah so mostly I went for the specific execution cycle because the docs mentioned a small performance downgrade when using this setting. maybe it wasn't worth it.We can change it back to init or to ansible if you think that's the best way to go. 
https://secure.php.net/manual/en/filesystem.configuration.php#ini.auto-detect-line-endings

Reply:
Now that I think about it, it's probably not going to affect `most` of the app tho, because it'll only really affects us in fgets/file related execution flows..
So, this might have been a silly thing to do.